### DISCUSSION TITLE
arbitrary values for box shadow

### DISCUSSION THREAD

**Discussion by pincman** [windicss/windicss#621] [General]
follow document [https://tailwindcss.com/docs/box-shadow](https://tailwindcss.com/docs/box-shadow)
`shadow-[0_35px_60px_-15px_rgba(0,0,0,0.3)]` or `shadow-inset-[0_35px_60px_-15px_rgba(0,0,0,0.3)]`
not work

**Comment by jiblett1000** [comment]
Wondering the same thing. Can't find a solution to this.

**Reply by donaldG21** [reply]
> Wondering the same thing. Can't find a solution to this.

Ditto. Spent too many hours, more than I'd like to admit, debugging a background image not working. And now shadow arbitrary values as well
